### ISSUE TITLE
Cannot install to project with latest Xamarin.Android.Support library

### ISSUE THREAD

**Issue by jamesmcroft** [AzureAD/azure-activedirectory-library-for-dotnet#1486]
**Which Version of ADAL are you using ?**
ADAL 4.5.0

**Which platform has the issue?**
xamarin android

**Repro**

Try to install the Microsoft.IdentityModel.Clients.ActiveDirectory 4.5.0 NuGet package to an Android project which has the expected latest Xamarin.Android.Support libraries installed (version 28.x). 

**Expected behavior**
The NuGet package should install successfully.

**Actual behavior**
The NuGet package fails to install due to a conflict in the version of the Xamarin.Android.Support library versions.

Output error reads:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1107	Version conflict detected for Xamarin.Android.Support.Compat. Install/reference Xamarin.Android.Support.Compat 28.0.0.1 directly to project Sample.Android to resolve this issue. 
 Sample.Android -> Xamarin.Android.Support.v4 28.0.0.1 -> Xamarin.Android.Support.Compat (= 28.0.0.1) 
 Sample.Android -> Microsoft.IdentityModel.Clients.ActiveDirectory 4.5.0 -> Xamarin.Android.Support.CustomTabs 27.0.2 -> Xamarin.Android.Support.Compat (= 27.0.2).	Sample.Android	D:\S\T\Security\Sample.Android\Sample.Android.csproj	1	
```

**Possible Solution**
Update version of Xamarin.Android.Support libraries to be the latest release or alter the project to not rely on the dependency.

**Comment by jamesmcroft** [comment]
I've created a PR for this issue but I noticed that the current target version is Android 8.1 which only supports a maximum Xamarin.Android.Support library of 27.0.2.1.

I've revved them to this version but should there be a consideration to move to Android 9.0 and the 28.x libraries?

**Comment by henrik-me** [comment]
Fix has been merged. Will be part of the next release of ADAL.

**Comment by jamesmcroft** [comment]
@henrik-me thanks for merging this fix.

**Comment by ghost** [comment]
When will a build of this be available as pre-release or a full release?  The requirements of 27.0.2 instead of 27.0.2.1. or 28.0.0.1 is not allowing me to include the ADAL library which in turn is blocking all Android development of a project I'm working on.

**Comment by jmprieur** [comment]
Released in ADAL 4.5.1

**Comment by jmprieur** [comment]
Released in ADAL 4.5.1
